src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -174,6 +175,13 @@ class editIndex extends viewModelBase {
             this.$previewEditor = $(editIndex.previewEditorSelector);
             this.previewEditor = aceEditorBindingHandler.getEditorBySelection(this.$previewEditor);
             this.previewEditor.setOption("wrap", true);
+
+            setTimeout(() => this.initRollingTooltip(), 0);
```

Comment:
why 2 separate setTimeouts? Can't we group that into single?

Reply:
fixed